Makefile

```diff
@@ -84,6 +84,11 @@ run: ## start the development server using Docker
 	@$(COMPOSE_RUN) dockerize -wait tcp://db:5432 -timeout 60s
 .PHONY: run
 
+ngrok: run ## start the development server using Docker through ngrok
+	@$(COMPOSE) up -d ngrok
+	@bin/set_ngrok_url
```

Comment:
I still think a task should be added in `src/aws/`. This is where live all task linked to aws.

Reply:
Indeed, if we choose to remove terraform apply from main makefile, it will go there.

Reply:
No this is definitely not a good idea.